### DISCUSSION TITLE
[Q] Listen external queue in RabbitMQ

### DISCUSSION THREAD

**Discussion by Colomix** [roadrunner-server/roadrunner#1113] [Q&A] [answered]
Hello!
I have integration with external system via rabbitmq. Messages are published by external system in fanout exchange for all system include me, because messages don't contain headers rr_id and rr_job. Without them listener amqp throw exception.
How to read the queue using roadrunner's jobs?

**Comment by rustatian** [comment, marked as answer]
Hey @Colomix 👋🏻. RR accepts only properly formed messages (`Jobs`). We have a ticket: https://github.com/roadrunner-server/roadrunner/issues/979 to have `raw` drivers which can read and process any payload. You may vote for it, and we'll include it in the next sprint.
